content/tutorials/map_algebra/basic_map_algebra.qmd

```diff
@@ -0,0 +1,511 @@
+categories: [map algebra, raster, beginner]
+description: Learn the basics of map algebra in GRASS.
+thumbnail: images/map_algebra_00.webp
+format:
+  ipynb: default
+  html:
+    toc: true
+    code-tools: true
+    code-copy: true
+    code-fold: false
+
+engine: jupyter
+execute:
+  eval: false
+jupyter: python3
+---
+
+![Terrain modeled with map algebra](images/map_algebra_00.webp)
+
+::: {.callout-note title="Computational notebook"}
+This tutorial can be run as a 
+[computational notebook](https://grass-tutorials.osgeo.org/content/tutorials/map_algebra/basic_map_algebra.ipynb).
+Learn how to work with notebooks with the tutorial
+[Get started with GRASS & Python in Jupyter Notebooks](../get_started/fast_track_grass_and_python.qmd).
+:::
+
+# Map Algebra
+This tutorial is a gentle introduction 
+to the basics of map algebra in GRASS. 
+You will begin to learn how to think procedurally 
+about space using math, logic, and flow control. 
+Map algebra is an algebraic approach
+to modeling fields of data such as rasters.
+It can be used to create, summarize, transform, 
+or combine multiple rasters. 
+Map algebra is powerful; 
+it can be used to model natural and anthropogenic phenomena 
+in space and even time. 
+Algebraic operations on raster maps can be classified 
+as local, focal, zonal, or global.
+Local operations act on each raster cell in isolation.
+Focal operations act on a neighborhood around each cell.
+Zonal operations act on groups of cells. 
+Global operations act on all cells in the raster.
+While there are many tools for map algebra in GRASS, 
+this tutorial focuses on local algebraic operations 
+using the raster map calculator
+[r.mapcalc](https://grass.osgeo.org/grass-stable/manuals/r.mapcalc.html). 
+It also briefly introduces focal operations with
+[r.neighbors](https://grass.osgeo.org/grass-stable/manuals/r.neighbors.html),
+zonal operations using logic and flow control with
+[r.mapcalc](https://grass.osgeo.org/grass-stable/manuals/r.mapcalc.html),
+and global operations with
+[r.info](https://grass.osgeo.org/grass-stable/manuals/r.info.html). 
+
+The raster map calculator
+[r.mapcalc](https://grass.osgeo.org/grass-stable/manuals/r.mapcalc.html)
+uses map algebra expressions to transform rasters. 
+The left side of the expression is the output raster map,
+while the right side is the input syntax including
+rasters, operators, functions, and variables.
+The raster calculator can be used for the simplest of tasks such as 
+copying rasters, creating constant rasters, scaling rasters,
+patching rasters together, or adding rasters together.
+It can also be used more sophisticated tasks such as 
+normalizing rasters,
+processing imagery,
+calculating vegetation indices,
+implementing mathematical models,
+and generating synthetic data. 
+In this tutorial, we will use map algebra to generate terrain
+with mountains, river valleys, plateaus, and forests. 
+We will use the following operators, functions, and internal variables:
```

Comment:
I would break this text into paragraphs or bullets as appropriate, to highlight the concepts and make it more readable.